back/src/modules/products/model/products.model.ts

```diff
@@ -45,6 +45,14 @@ export class Product {
   @Field(() => ProductCharacteristics, { nullable: true })
   @Prop({ default: {} })
   characteristics: ProductCharacteristics;
+
+  @Field(() => Float)
+  @Prop()
+  price: number;
+
+  @Field(() => Int)
+  @Prop()
+  percentDiscount: number;
```

Comment:
**Suggestion:** The 'percentDiscount' field is declared as an Int in the GraphQL schema, but it's more appropriate to use Float for percentage values to allow for fractions.
```suggestion
  @Field(() => Float)
  @Prop()
  percentDiscount: number;
```